packages/cli/src/ErrorReporting.ts

```diff
@@ -12,33 +12,65 @@ export const initErrorHandling = async () => {
 	});
 
 	const dsn = config.getEnv('diagnostics.config.sentry.dsn');
-	if (!config.getEnv('diagnostics.enabled') || !dsn) {
+	if (!dsn) {
 		initialized = true;
 		return;
 	}
 
 	// Collect longer stacktraces
 	Error.stackTraceLimit = 50;
 
-	const { N8N_VERSION: release, ENVIRONMENT: environment } = process.env;
+	const {
+		N8N_VERSION: release,
+		ENVIRONMENT: environment,
+		DEPLOYMENT_NAME: serverName,
+	} = process.env;
 
-	const { init, captureException, addGlobalEventProcessor } = await import('@sentry/node');
+	const { init, captureException, addEventProcessor } = await import('@sentry/node');
 
 	const { RewriteFrames } = await import('@sentry/integrations');
+	const { Integrations } = await import('@sentry/node');
 
+	const blockedIntegrations = [
+		'Console',
+		'Http',
+		'Undici',
+		'OnUncaughtException',
+		'LocalVariables',
+		'Context',
+		'Modules',
+		'RequestData',
+	];
```

Comment:
Since we are blocking most of the integrations, would it make more sense to specify the ones we are actually using? Would make it also much clearer to see what we have in use